source/fundamentals/data-formats/bson.txt

```diff
@@ -0,0 +1,224 @@
+.. _csharp-bson:
+
+==============
+Work with BSON
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to create BSON documents, read BSON from a file, 
+and write BSON to a file using the {+driver-short+}.
+
+BSON Data Format
+----------------
+
+**BSON**, or Binary JSON, is the data format that MongoDB uses to organize
+and store data. This data format includes all JSON data structure types and
+adds support for types including dates, different size integers, ObjectIds, and
+binary data. For a complete list of supported types, see the
+:manual:`BSON Types </reference/bson-types>` server manual page.
+
+The code samples in this guide use the following BSON document as an example:
+
+.. code-block:: none
+
+   {
+       "address" : {
+           "street" : "Pizza St", 
+           "zipcode" : "10003" 
+       },
+       "coord" : [-73.982419, 41.579505]
+       "cuisine" : "Pizza",
+       "name" : "Mongo's Pizza"
+   }
+
+Create a BSON Document
+----------------------
+
+To build a BSON document in {+language+}, create an instance of the ``BsonDocument`` class.
+The ``BsonDocument`` constructor accepts ``BsonElement`` arguments that map to the fields
+and values in the document. You can either construct individual ``BsonElement`` objects 
+and pass them to the ``BsonDocument`` constructor or pass field-value pairs inside curly 
+braces ( ``{}`` ). 
+
+The following code sample constructs a ``BsonDocument`` object to represent the previous
+BSON document:
+
+.. literalinclude:: ../../includes/fundamentals/code-examples/Bson.cs
+   :start-after: start-create
+   :end-before: end-create
+   :language: csharp
+   :copyable:
+   :dedent:
+
+.. tip::
+   
+   Each pair of curly braces in the BSON object requires one new ``BsonDocument`` object
+   in {+language+}.
```

Comment:
I'm a little confused by this tip and what it's referring to in general. Is it referring to curly braces in the BSON document shown in the first code example? If so maybe change it to "Each pair of curly braces in a BSON document" since it's referred to as document rather than object earlier.

Reply:
Yeah, that was my intention. You're exactly right about making it clearer.